src/auth.module.ts

```diff
@@ -35,7 +35,9 @@ export class AuthModule {
         },
         {
           provide: JWT_MAPPER,
-          useValue: options.jwtMapper ? options.jwtMapper : (payload) => payload,
```

Comment:
@jhaltercmu2020 lets not add stylistic changes to this commit.

Reply:
@jspizziri Fixed.